### ISSUE TITLE
How can I send message to slack channel indicate the bot is typing?

### ISSUE THREAD

**Issue by scyforce** [rampatra/jbot#158]
I am building an integration between and other services, when the integration get notified the agent from other service is typing, I want to send user_typing event to slack channel, so in the channel it will display bubble, I checked the doc this user_typing event is only available through RTM api, https://api.slack.com/events/user_typing

I wonder does JBOT provide this feature to directly using RTM client to interact with slack?

**Comment by rampatra** [comment]
Yes, it does. You can simply call `reply()` method with type `me.ramswaroop.jbot.core.common.EventType#USER_TYPING` like:
```
Message m = new Message();
m.setType(EventType.USER_TYPING.name().toLowerCase());
reply(session, event, m);
```
